activerecord/lib/active_record/railties/databases.rake

```diff
@@ -254,7 +254,7 @@ db_namespace = namespace :db do
       end
 
       desc 'Clear a db/schema_cache.dump file.'
-      task :clear => [:environment, :load_config] do
+      task :clear => [:load_config] do
         filename = File.join(ActiveRecord::Tasks::DatabaseTasks.db_dir, "schema_cache.dump")
```

Comment:
db_dir needs `Rails.application` https://github.com/square/rails/blob/dont-load-app-on-structure/activerecord/lib/active_record/tasks/database_tasks.rb#L53-L55 
Not sure if it is going to work without loading `environment`